### PR TITLE
feat(gestionnaire): Allow update of values

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

**Comment by nkokla** [comment]
@desoindx 
It seems this commit is include in another PR : 
https://github.com/betagouv/france-chaleur-urbaine/pull/288

I have review this other PR, so this PR is probably useless ?

File: src/components/Manager/AdditionalInformation.tsx [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+  const [value, setValue] = useState('');
+  useEffect(() => {
+    if (demand && value === '') {
+      if (demand[`Gestionnaire ${field}`] !== undefined) {
+        setValue(demand[`Gestionnaire ${field}`].toString());
+      } else if (demand[field]) {
+        setValue(demand[field].toString());
+      }
+    }
+  }, [demand, field]);
```
Comment (nkokla) [inline src/components/Manager/AdditionalInformation.tsx:25]: Like notify by github action : React Hook useEffect has a missing dependency: 'value'.

File: src/components/Manager/AdditionalInformation.tsx [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+        (e) =>
+          updateDemand(demand.id, {
+            [`Gestionnaire ${field}`]: parseFloat(e.target.value),
+          }),
+        500
+      ),
+    [demand.id, updateDemand, field]
+  );
+
+  useEffect(() => () => onChangeHandler.cancel(), [onChangeHandler]);
```
Comment (nkokla) [inline src/components/Manager/AdditionalInformation.tsx:39]: Idea : We probably should create our hook for the debounce for do not recreate this cancel.

File: src/components/Manager/AdditionalInformation.tsx [hunk 3]
```diff
@@ -1,11 +1,53 @@
...
+
   return (
     <Container>
-      <div>{demand.Logement && `${demand.Logement} lots`}</div>
-      <div>{demand.Conso && `${demand.Conso} MWh`}</div>
+      <TextInput
+        type="number"
+        value={value}
+        onChange={(e) => {
```
Comment (nkokla) [inline src/components/Manager/AdditionalInformation.tsx:46]: Idea: This handler function should be in a useCallback for escape useless render.

**Review by nkokla — CHANGES_REQUESTED** [review]

File: src/components/Manager/AdditionalInformation.tsx (hunk 1 above)
Comment (desoindx) [reply src/components/Manager/AdditionalInformation.tsx:25]: Ok with the warning

**Review by nkokla — APPROVED** [review]

**Pull request merged**